mods/d2k/rules/structures.yaml

```diff
@@ -883,14 +882,13 @@ high_tech_factory:
 		SpawnOffset: 0,0,728
 		ExitCell: 0,0
 	Building:
-		Footprint: _x_ xxx xxx
+		Footprint: _u_ xxx uuu
```

Comment:
Both this and the Research Center define a custom, slightly lowered offset for the top cell, so this needs to be `u` otherwise the main HitShape would add another targetable pos at the cell center.